rickshaw-run

```diff
@@ -1981,6 +1997,31 @@ sub load_settings_info() {
         }
     }
 
+    my $tmp_settings_value;
+    ($rc, $tmp_settings_value) = get_json_setting("engines.public.url", $registries_settings);
+    if ($rc != 0) {
+        print "load_settings_info(): failed to load public engines repo url\n";
+        exit 1;
+    } else {
+        $run{'reg-repo'} = $tmp_settings_value;
+        printf "load_settings_info(): loaded registry repository: %s\n", $run{'reg-repo'};
+    }
+
+    ($rc, $tmp_settings_value) = get_json_setting("engines.public.push-token", $registries_settings);
+    if ($rc != 0) {
+        print "load_settings_info(): failed to load public engines push-token\n";
+        exit 1;
+    } else {
+        $run{'reg-auth'} = $tmp_settings_value;
+        printf "load_settings_info(): loaded registry authorization token: %s\n", $run{'reg-auth'};
```

Comment:
this kind of info is more a debug level log, do we really want to print all these??

Reply:
I think that I want it in while we go through these transitions to aide in debugging anything that might go wrong, but I may eventually decide to pull it back out for the reasons you state.